src/DiffViewerProvider.ts

```diff
@@ -62,6 +63,7 @@ export class DiffViewerProvider implements vscode.CustomTextEditorProvider {
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.2.0/styles/github.min.css" />
         <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/diff2html/bundles/css/diff2html.min.css" />
         <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/diff2html/bundles/js/diff2html-ui.min.js"></script>
+				<link href="${tweaksCssUri}" rel="stylesheet" />
```

Comment:
Mind to use spaces instead of tabs?

I'll adjust the format script later to catch things like that.

Reply:
oh, this is unexpected! It turns out that in the original file the appCssUri link line has tabs, too; I copied the line and didn't notice I was copying tabs. I'll fix both lines.

Reply:
Since that was inside a string, prettyjs may need to ignore it no matter what settings it's given.

Reply:
Yep, I'll check that later.